main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/sirupsen/logrus"
+	"github.com/skynetlabs/pinner/api"
+	"github.com/skynetlabs/pinner/conf"
+	"github.com/skynetlabs/pinner/database"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+func main() {
+	// Load the configuration from the environment and the local .env file.
+	err := conf.LoadConfiguration()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Initialise the global context and logger. These will be used throughout
+	// the service. Once the context is closed, any background threads will
+	// wind themselves down.
+	ctx := context.Background()
+	logger := logrus.New()
+	logLevel, err := logrus.ParseLevel(conf.LogLevel)
+	if err != nil {
+		logLevel = logrus.InfoLevel
```

Comment:
Hm, good point. Initially, this was done to accommodate an empty value in the config because it's not an obligatory conf value. But now that we have a decent configuration initialisation, any error here will be caused by the config file and we should die on it.